src/Editor/index.jsx

```diff
@@ -4,6 +4,8 @@ import AceEditor from "react-ace";
 import { UndoManager } from "brace";
 import "brace/theme/idle_fingers";
 import "brace/ext/searchbox";
+import "ace-builds/src-noconflict/keybinding-vim";
+import "ace-builds/src-noconflict/keybinding-emacs";
```

Comment:
added the import for emacs keybindings from `ace-builds`, so now it should actually work